src/components/footer/FooterLinks.js

```diff
@@ -13,11 +12,11 @@ const FooterLinks = () => {
         render={data => {
           return (
             <NavMenu
-              as={Flex}
-              w={['90%', '90%', '90%', 'full']}
-              justify="space-evenly"
-              wrap={['wrap', 'wrap', 'wrap', null]}
-              align="center"
+              style={{
```

Comment:
@racedale did you try doing `as={List}` instead of `as={Flex}`

At that point we should be able to do `justifyContent='space-evenly'` and `w="100%"` as props.

I think. Still not 100% solid with my Chakra skills.

Reply:
Yeah tried that, `NavMenu` is a List component by default which is why I didn't do `as={List}`, but it [doesn't support any of the flex box props](https://chakra-ui.com/list#props) which makes sense since it's not the Flex component anymore